contracts/src/v0.8/vrf/dev/VRFV2PlusWrapper.sol

```diff
@@ -221,6 +228,18 @@ contract VRFV2PlusWrapper is ConfirmedOwner, TypeAndVersionInterface, VRFConsume
     s_fallbackWeiPerUnitLink = _fallbackWeiPerUnitLink;
     s_fulfillmentFlatFeeLinkPPM = _fulfillmentFlatFeeLinkPPM;
     s_fulfillmentFlatFeeNativePPM = _fulfillmentFlatFeeNativePPM;
+
+    emit ConfigSet(
+      _wrapperGasOverhead,
+      _coordinatorGasOverhead,
+      _wrapperPremiumPercentage,
+      _keyHash,
+      _maxNumWords,
+      _stalenessSeconds,
+      _fallbackWeiPerUnitLink,
+      _fulfillmentFlatFeeLinkPPM,
+      _fulfillmentFlatFeeNativePPM
+    );
```

Comment:
Just curious, why emit this event if we have a view function where we should be able to check all the parameters that have been previously set?

Reply:
It's for people to listen if the config has changed. Imagine if there's no such event emitted and you want to know whether the config has been changed. Then you would need to poll the blockchain every X seconds to check if the value has changed.